replayserver/bookkeeping/database.py

```diff
@@ -157,6 +158,15 @@ def __init__(self, db):
             'num_players': player_count
         }
 
+    async def update_game_stats(self, game_id, replay_ticks):
```

Comment:
Is there a reason for putting this info in the db instead of the replay header? Is this meant to be a way to communicate with another service?

Reply:
The whole idea of this is so that the actual in game times can be displayed in the vault (see the DB PR that I linked above). This way it can be queried by the API.